superset/__init__.py

```diff
@@ -209,15 +210,20 @@ def index(self):
 results_backend = app.config.get('RESULTS_BACKEND')
 
 # Merge user defined feature flags with default feature flags
-feature_flags = app.config.get('DEFAULT_FEATURE_FLAGS')
-feature_flags.update(app.config.get('FEATURE_FLAGS') or {})
+_feature_flags = app.config.get('DEFAULT_FEATURE_FLAGS')
```

Comment:
Can we just go `app.config.get('DEFAULT_FEATURE_FLAGS', {})` if the key doesnt exist the next param (`{}`) will be set to `_feature_flag`